src/wallet/cc_wallet/debug_spend_bundle.py

```diff
@@ -48,8 +48,6 @@ def debug_spend_bundle(spend_bundle: SpendBundle) -> None:
     its clvm.
     """
 
```

Comment:
Probably want to add `assert_announcement_consumed` set thing here to make this tool more useful.

Reply:
added announcements created